Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:stretch
+
+#TODO: Verify Go version for mender 2.0.0 beta release
+ARG MENDER_VERSION=2.0.0-beta
+ARG GOLANG_VERSION=1.11.5
+
+RUN apt-get update && apt-get install -y \
+    git wget \
+    build-essential debhelper devscripts \
```

Comment:
I am working on removing some dependencies (i.e. find the actual tools we need and not all these meta-packages)

Reply:
Ok, I will leave it as it is.